src/scheme.mli

```diff
@@ -3,32 +3,32 @@
 open! Stdune
 
 type 'rules t =
-  (** [Empty] is a scheme that has no rules *)
   | Empty
+  (** [Empty] is a scheme that has no rules *)
```

Comment:
The compiler now complains if the comment is before the constructor, so I moved them all after